src/arm/BBORG_SERVO-00A2.dts

```diff
@@ -75,45 +75,27 @@
     #address-cells = <1>;
     #size-cells = <0>;
 
-    pca9685_servo: pca9685_serve@7f {
-        compatible = "nxp, pca9685";
+    pca: pca@70 {
+        compatible = "nxp,pca9685-pwm";
+        #pwm-cells = <16>;
+        reg = <0x70>;
         lable = "pca9685_servo";
-        pinctrl-names = "default";
-        pinctrl-0 = <&bborg_servo_pca9685_en_pins>;
-        enable-gpios = <&gpio2 4 GPIO_ACTIVE_HIGH>;
-        reg = <0x7f>;
+        /* invert; */
+        /* open-drain; */
     };
 };
 
 &{/} {
-    gpio_inputs {
-        compatible = "gpio-keys";
+    leds {
         pinctrl-names = "default";
-        pinctrl-0 = <&bborg_servo_input_pins>;
+        pinctrl-0 = <&bborg_servo_pca9685_en_pins>;
 
-        input@1 {
-            lable = "in1";
-            gpios = <&gpio1 12 GPIO_ACTIVE_HIGH>;
-        };
-        input@2 {
-            lable = "in2";
-            gpios = <&gpio1 13 GPIO_ACTIVE_HIGH>;
-        };
-        input@3 {
-            lable = "in3";
-            gpios = <&gpio3 16 GPIO_ACTIVE_HIGH>;
-        };
-        input@4 {
-            lable = "in4";
-            gpios = <&gpio3 19 GPIO_ACTIVE_HIGH>;
-        };
-        input@5 {
-            lable = "in5";
-            gpios = <&gpio3 20 GPIO_ACTIVE_HIGH>;
-        };
-        input@6 {
-            lable = "in6";
-            gpios = <&gpio0 7 GPIO_ACTIVE_HIGH>;
+        compatible = "gpio-leds";
+
+        pca9685-enable {
```

Comment:
This is just an enable GPIO, right? There's no real LED, is there? Would be good to clarify in the comments.

Reply:
no there is not really any led on the cape, this pin control the PCA9685's enable pin.